frontend/src/components/EOI/ExpressionOfInterestForm.vue

```diff
@@ -1,96 +1,306 @@
-        @submit.prevent="handleSubmit"
-        id="expressionOfInterestForm"
-        ref="expressionOfInterestForm"
-        validate-on="blur"
-        v-model="isFormValid"
-      >
-        <v-container fluid class="content-container">
-          <div class="form-container">
-            <div class="d-flex justify-space-between">
-              <h1>Expression of Interest</h1>
-            </div>
-            <br />
+  <ConfirmationDialog ref="confirmDelete">
+    <template #message>
+      <p>
+        The Expression of Interest application will be removed from your
+        records. A new EOI can be submitted in the future.
+      </p>
+    </template>
+  </ConfirmationDialog>
+  <ConfirmationDialog ref="confirmDeleteDocument">
+    <template #message>
+      <p>This document will be removed from your records.</p>
+    </template>
+  </ConfirmationDialog>
+  <v-form
+    @submit.prevent="handleSubmit"
+    id="expressionOfInterestForm"
+    ref="expressionOfInterestForm"
+    validate-on="blur"
+    v-model="isFormValid"
+  >
+    <v-container fluid>
+      <div>
+        <div class="d-flex justify-space-between">
+          <h1>Expression of Interest</h1>
+        </div>
+        <IndependentSchoolDisclaimer />
+        <br />
+        <v-divider></v-divider>
+        <div>
+          <div v-if="!isNew && data">
+            <EOIFormHeader :eoi="data" :draftStatusCode="draftStatusCode" />
             <v-divider></v-divider>
-            <div v-if="!isEditing && !isNew()">
-              <ExpressionOfInterestReadOnlyView :eoi="this.eoi" />
-            </div>
-            <div v-else>
-              <div v-if="!isNew()">
-                <h4>General</h4>
-                <br />
-                <v-row>
-                  <v-col cols="12" sm="12" md="6" xs="12">
-                    <v-label>EOI Number </v-label>
-                    <p>{{ data.iosas_eionumber }}</p>
-                  </v-col>
-                  <v-col cols="12" sm="12" md="6" xs="12">
-                    <v-label>Status </v-label>
-                    <p>{{ data.iosas_reviewstatus }}</p>
-                  </v-col>
-                </v-row>
-                <v-divider></v-divider>
-              </div>
-              <h4>School Authority Information</h4>
-              <v-row>
-                <v-col cols="12" sm="12" md="6" xs="12">
-                  <v-label
-                    >Name of School Authority (as it appears on attached
-                    incorporation documents)</v-label
-                  >
+          </div>
+          <h4>School Authority Information</h4>
+          <br />
+          <v-row>
+            <v-col cols="12">
+              <v-label class="no-mb"
+                >Is this Expression of Interest for an existing School
+                Authority?</v-label
+              >
+              <v-radio-group
+                id="iosas_existingauthority"
+                v-model="data.iosas_existingauthority"
+                color="#003366"
```

Comment:
Can we manage this with style class only?

Reply:
I have not found a way to override the vuetify class for the inputs, This color prop changes the active border style all inputs and was the existing pattern on the app.